### ISSUE TITLE
User supplied `WINEDLLPATH` discarded by proton

### ISSUE THREAD

**Issue by namaenonaimumei** [ValveSoftware/Proton#7141]
Version in question: 8.0 (but all versions seem the same)

Changing [the culprit line](https://github.com/ValveSoftware/Proton/blob/c32e9213c657d163c557475ff3249acdeb7debeb/proton#L1241) to below results in behaviour consistent with that of wine.
```python
self.env["WINEDLLPATH"] = g_proton.lib64_dir + "/wine:" + g_proton.lib_dir + "/wine"
if len(os.getenv("WINEDLLPATH") or ""):
    self.env["WINEDLLPATH"] += ":" + os.getenv("WINEDLLPATH").strip(":")
```

Couldn't find any mentions of current behaviour being intentional, so considering this as a regression on proton's side.

**Comment by namaenonaimumei** [comment]
Any official statement whether this behaviour is intended?
Overriding this variable on proton's side disallows user from providing additional winelibs and forces usage of regsvr32, which is less flexible

**Comment by kitlith** [comment]
This appears to have been fixed in Proton 11: <https://github.com/ValveSoftware/Proton/blob/7a3a82e05d3a0813de769b6ce7077b6b23fb1c39/proton#L1522-L1524>

**Comment by namaenonaimumei** [comment]
Right, sorry for the delay, I was actually applying the similar patch up until somewhere around proton-ge 10, where it was addressed (not sure about the exact version). I can confirm that vanilla proton 11 now matches Wine behaviour for `WINEDLLPATH`.
Thanks for upstreaming this.
